src/main.rs

```diff
@@ -858,6 +859,21 @@ mod tests {
         Parser::parse(contents.into().as_bytes()).unwrap()
     }
 
+    // Parse a string of code that is meant to fail
+    fn parse_str_fail(contents: impl Into<String>) -> Parser {
+        // Parse the code
+        let res = Parser::parse(contents.into().as_bytes());
+        // If the code parsed without error
+        if res.is_ok() {
+            // There is a problem with the parser
+            // Output the result in an error (failing the test)
+            panic!("{:?}", res);
```

Comment:
If we use `if let Ok(...)` above we can change this to the variable inside the `Ok(..)`
```suggestion
            panic!("{:?}", parsed_content);
```